### PR TITLE
Module labeling

### PR DESCRIPTION
**Pull request by comeonbuddy**
### Please check if the PR fulfills these requirements
- [ ] The commit message follows our Submission guidelines
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

### What kind of change does this PR introduce? 
(Bug fix, feature, docs update, ...)

### What is the current behavior? 
(You can also link to an open issue here)

### What is the new behavior?
(Define and describe any new functionality. Clarify if this is a feature change)

### Does this PR introduce a breaking change?
(What changes might users need to make in their application due to this PR?)

### CODE REVIEW

File: contracts/tokens/SecurityToken.sol [hunk 1]
```diff
@@ -213,21 +214,24 @@ contract SecurityToken is ERC20, ERC20Detailed, ReentrancyGuard, RegistryUpdater
...
+
+    function addModuleWithLabel(
         address _moduleFactory,
         bytes _data,
         uint256 _maxCost,
-        uint256 _budget
-    ) external onlyOwner nonReentrant {
+        uint256 _budget, 
+        bytes32 _label
+    ) onlyOwner nonReentrant {
```
Comment (satyamakgec) [inline contracts/tokens/SecurityToken.sol]: add `public` explicitly

File: contracts/tokens/SecurityToken.sol [hunk 2]
```diff
@@ -250,12 +254,24 @@ contract SecurityToken is ERC20, ERC20Detailed, ReentrancyGuard, RegistryUpdater
...
+    /**
+    * @notice addModule function will call addModuleWithLabel() with an empty label for backward compatible
+    */
+    function addModule(
+        address _moduleFactory,
+        bytes _data,
+        uint256 _maxCost,
+        uint256 _budget
+    ) external { 
+        return addModuleWithLabel(_moduleFactory, _data, _maxCost, _budget, "");
```
Comment (satyamakgec) [inline contracts/tokens/SecurityToken.sol]: no need of `return`

**Review by satyamakgec — CHANGES_REQUESTED** [review]

File: contracts/tokens/SecurityToken.sol (hunk 1 above)
Comment (comeonbuddy) [reply contracts/tokens/SecurityToken.sol]: done

File: contracts/tokens/SecurityToken.sol (hunk 2 above)
Comment (comeonbuddy) [reply contracts/tokens/SecurityToken.sol]: done

**Review by satyamakgec — APPROVED** [review]

**Pull request merged**